auth-api/src/auth_api/schemas/schemas/business.json

```diff
@@ -8,7 +8,11 @@
     "examples": [
         {
             "affiliations": [
-                1
+                {
+                    "id": 1,
```

Comment:
Is this the id of the user or of the affiliation?

Reply:
id of the affiliation